test/simplifyExpression/oneStep.test.js

```diff
@@ -6,28 +6,28 @@ const ChangeTypes = require('../../lib/ChangeTypes');
 const simplifyExpression = require('../../lib/simplifyExpression');
 
 function testOneStep(exprStr, outputStr, debug=false) {
-  const steps = simplifyExpression(exprStr);
-  if (!steps.length) {
-    return exprStr;
-  }
-  const nodeStatus = steps[0];
-  if (debug) {
-    if (!nodeStatus.changeType) {
-      throw Error('missing or bad change type');
-    }
-    // eslint-disable-next-line
-    console.log(nodeStatus.changeType);
-    // eslint-disable-next-line
-    console.log(print(nodeStatus.newNode));
-  }
```

Comment:
Code that's outside of an `it` block will get run regardless so whether the `describe` is disabled (via .skip) or not.